backend/Moenda/MD/MDRules.test.js

```diff
@@ -0,0 +1,66 @@
+const mdRoules = require("./MDRules");
+
+// FirstSectionStartsWithHx
+
+test('ruleFirstSectionStartsWithHx returns object', 
+  () => {
+    const object = mdRoules.ruleFirstSectionStartsWithHx("/home/felipe/fork/Moenda/README.md", "/home/felipe/fork/Moenda/config.js");
+    expect("object").toBe(typeof object);
```

Comment:
Is not better to compare the content instead of the types? What are the advantages doing that?